install.sh

```diff
@@ -358,6 +358,9 @@ else
 
   # Make the git default branch error messages go away.
   git config --global init.defaultBranch main
+  # Help handle situations where GitHub is down.
+  git config --global http.lowSpeedLimit 1000
```

Comment:
If possible we should avoid setting global configuration settings-- can we set these in `CREW_PREFIX/lib/crew` instead?

Reply:
Sure.